src/Integration.Vsix/CFamily/CFamilyHelper.cs

```diff
@@ -267,11 +267,14 @@ internal static FileConfig TryGetConfig(ILogger logger, ProjectItem projectItem,
 
             try
             {
+                // Note: if the C++ tools are not installed then it's likely an exception will be thrown when
+                // the framework tries to JIT-compile the TryGet method (since it won't be able to find the MS.VS.VCProjectEngine
+                // types).
```

Comment:
The previous version of the code would have failed too if the C++ tools weren't installed, but the error type would have been different.